test/dlc_tests/benchmark/sagemaker/mxnet/training/resources/scripts/smtrain-resnet50-imagenet.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+# PreRequisities
+# Install mxnet >=1.3b and gluoncv on each machine
+# Ensure data is kept at ~/data for each machine or change the data paths below
+# Example command to start the training job
+# Specify hosts in the file `hosts`
```

Comment:
Is this still relevant?

Reply:
While we aren't using training data, one can choose to provide imagenet data as input for the training script. In that case it will be helpful to know where does data go.

For test purposes, we are using synthetic data and we dont need to specify hosts as we leverage sagemaker training.

I'd leave these things as is, because they are true for running this individual file on its own.

Reply:
Sure. Thanks for the explanation!